categories/README.md

```diff
@@ -1,51 +1,57 @@
 API to create and get Joomla categories
 
-## Create / Update Category
+## Get Single Category
 
 ```http
-POST /index.php?option=com_api&app=categories&resource=categories&format=raw
+GET /index.php?option=com_api&app=categories&resource=category&format=raw&id=:id
 ```
-OR update an existing category
+
+Alternatively, you can call this API as follow (assuming you have created a menu with alias as 'api' for com_api)
+
 ```http
-POST /index.php?option=com_api&app=categories&resource=categories&format=raw&id=:id
+GET /api/categories/category/:id
 ```
 
 #### Request Params
 
-| Param Name | Required | Type | Comment  |
-| ---------- | -------- | ------- | :---- |
-| title      | YES      | STRING |         |
-| alias      | NO      | STRING | URL alias. Will be generated based on title if kept empty |
-| description    | NO      | STRING |        |
-| published    | NO      | INT | 1 = Published (Default) / 0 = Unpublished |
-| parent_id      | NO      | INT |  Specify a parent category id if you wish to create a subcategory |
-| language | NO | STRING | Will use the site's default language if none provided. |
-| extension | YES | STRING | Since Joomla supports using the categories extension for 3rd party components, this field specifies the extension name. Use com_content for article categories.  |
-| access | NO | INT | Access Level for Category |
+| Param Name | Required | Comment |
+| ---------- | -------- | :------ |
+| fields         | NO       | Defaults to id, title, created_time |
 
 
 #### Response Params
 
-| Param Name | Comment |
-| ---------- | :------ |
-| success | true if the category was created, false if there was a problem |
+| Param Name | Comment  |
+| ---------- | :------- |
+| success | true if the API call succeeds, false if there was a problem |
 | message | Error mesage in case success is false |
 | data.results | Array containing a single [Category Object](#category-object) in case of success. Empty array in case of failure. |
 
 ## Get Categories List
+
 ```http
 GET /index.php?option=com_api&app=categories&resource=categories&format=raw
 ```
+
+Alternatively, you can call this API as follow (assuming you have created a menu with alias as 'api' for com_api)
+
+```http
+GET /api/categories/categories
```

Comment:
Can this be `/categories/all/:extension`. The value of extension will be available in the `id` param. Not ideal, but the non-sef URL becomes semantic.

Reply:
need router support here as well as described at https://github.com/techjoomla/com_api-plugins/pull/90/files#r245647296 @coolbung